scripts/properly_populate_transaction_fees.js

```diff
@@ -0,0 +1,281 @@
+/**
+ * These are some recommendations for using the script in
+ * *development* mode. We currently have ~70000 transactions in our
+ * database, running it with a large batch to process all the
+ * transactions at once is the fastest way to run it and get the
+ * output, however it also exhausts node's memory limits with batches
+ * larger than ~50000. Use --max-old-space=8192 to do it all in one
+ * batch and iterate faster.
+ *
+ * For production usage, It might be a good idea to break it down in
+ * multiple batches to leave some room for the database to process
+ * operations from the web clients. I've been running 7 batches of
+ * 10000 and didn't take more than ~20s in my machine (i7~2.2GHz),
+ * which is just a little bit slower than running it all in one batch.
```

Comment:
Nice to have these perf numbers!

Reply:
I started measuring because it exhausted the v8 memory when I joined the Collectives table to read the `hostFeePercent` but I think it also helps knowing what to expect when we run it on prod.